src/migrations/1621511823381_nft_metadata.ts

```diff
@@ -32,6 +32,7 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
     contract_id: {
       type: 'string', 
       notNull: true, 
+      unique: true,
```

Comment:
Only keeping `contract_id` unique does not work for NFTs because NFTs have multiple metadata entries (one per individual token).
The unique constraint should be with both `contract_id` and the token id (not yet stored in the table, you'll need to add that)